mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1426,6 +1437,10 @@
     <string name="etp_redirect_trackers_title">Redirect Trackers</string>
     <!-- Description of redirect tracker cookies that can be blocked by Enhanced Tracking Protection -->
     <string name="etp_redirect_trackers_description">Clears cookies set by redirects to known tracking websites.</string>
+    <!-- Description of the SmartBlock Enhanced Tracking Protection feature -->
+    <string name="preference_etp_smartblock_description">Some trackers marked below have been partially unblocked on this page because you interacted with them *.</string>
```

Comment:
Seems like there is an extra character not needed here, at the end of the string:  *

Reply:
The `*` character is expected, as it's what we are showing in the UI for users identify trackers. I will clarify the comments :)

Reply:
Oh ok! thanks, yes please do add clarification in comments :)

Reply:
Sorry for the delay,  I file a PR for updating the comments
https://github.com/mozilla-mobile/fenix/pull/21323/